src/main/java/mate/academy/spring/dto/DtoUtil.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.spring.dto;
+
+import mate.academy.spring.entity.User;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
@TatianaPirch Usually util classes are not components) but let we leave it as is